core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -144,6 +151,21 @@ public static int defaultBackoffMaxAttempts() {
         return DEFAULT_BACKOFF_MAX_ATTEMPTS;
     }
 
+    /**
+     * Returns the default value of the {@code defaultExponentialBackoffStartInterval} parameter when
+     * instantiating a {@link Backoff} using {@link Backoff#exponential()}. Note that this value
```

Comment:
- `{@code defaultExponential...}` -> `{@code minIntervalMillis`}`
- `{@link Backoff#exponential()}` -> `{@link Backoff#exponential(long, long)}` (I believe `{@link Backoff#exponential()}` will not work because of mismatching method signature. Am I missing something?)